doc/whats_new/v0.24.rst

```diff
@@ -70,6 +70,15 @@ Changelog
   `init_size_`, are deprecated and will be removed in 0.26. :pr:`17864` by
   :user:`Jérémie du Boisberranger <jeremiedbb>`.
 
+- |Fix| :class:`cluster.AgglomerativeClustering.__init__()` has a new parameter
```

Comment:
Here the link  [doesn't get generated properly](https://113729-843222-gh.circle-artifacts.com/0/doc/whats_new/v0.24.html) so I think we can keep
```suggestion
- |Fix| :class:`cluster.AgglomerativeClustering` has a new parameter
```